airflow/providers/sftp/sensors/sftp.py

```diff
@@ -71,6 +71,8 @@ def poke(self, context: 'Context') -> bool:
                 self.actual_file_to_check = file_from_pattern
             else:
                 return False
+        else:
+            self.actual_file_to_check = self.path
```

Comment:
Good catch! IMO it's probably not worth having `actual_file_to_check` be an instance attr since it's not used outside of the `poke()` method. WDYT? If you agree, it should be deleted from `__init__()` as well.